docs/uvdata_tutorial.rst

```diff
@@ -1021,1050 +1062,1030 @@ b) Select 3 antennas to keep using the antenna names, also select 5 frequencies
-  ['E09', 'W06', 'N02']
 
-  >>> # print all the frequencies after the select
-  >>> print(uvd.freq_array)
-  [3.6304542e+10 3.6304667e+10 3.6304792e+10 3.6304917e+10]
+UVData: Summing and differencing visibilities
+---------------------------------------------
+Simple summing and differencing of visibilities can be done with the :meth:`pyuvdata.UVData.sum_vis`
+and :meth:`pyuvdata.UVData.diff_vis` methods.
 
-c) Select a few antenna pairs to keep
-*************************************
 .. code-block:: python
 
   >>> import os
+  >>> from astropy.time import Time
   >>> from pyuvdata import UVData
   >>> from pyuvdata.data import DATA_PATH
   >>> filename = os.path.join(DATA_PATH, 'day2_TDEM0003_10s_norx_1src_1spw.uvfits')
-  >>> uvd = UVData.from_file(filename)
-
-  >>> # print how many antenna pairs with data in the original file
-  >>> print(len(set(zip(uvd.ant_1_array, uvd.ant_2_array))))
-  153
-  >>> uvd.select(bls=[(1, 2), (7, 1), (1, 21)])
-
-  >>> # note that order of the values in the pair does not matter
-  >>> # print all the antenna pairs after the select
-  >>> print(sorted(set(zip(uvd.ant_1_array.tolist(), uvd.ant_2_array.tolist()))))
-  [(1, 2), (1, 7), (1, 21)]
+  >>> uvd1 = UVData.from_file(filename)
+  >>> uvd2 = uvd1.copy()
 
-d) Select antenna pairs using baseline numbers
-**********************************************
-.. code-block:: python
+  >>> # sum visibilities
+  >>> uvd1 = uvd1.sum_vis(uvd2)
 
-  >>> import os
-  >>> import numpy as np
-  >>> from pyuvdata import UVData
-  >>> from pyuvdata.data import DATA_PATH
-  >>> filename = os.path.join(DATA_PATH, 'day2_TDEM0003_10s_norx_1src_1spw.uvfits')
-  >>> uvd = UVData.from_file(filename)
+  >>> # diff visibilities
+  >>> uvd1 = uvd1.diff_vis(uvd2)
 
-  >>> # baseline numbers can be found in the baseline_array
-  >>> print(len(uvd.baseline_array))
-  1360
+  >>> # in place option
+  >>> uvd1.sum_vis(uvd2, inplace=True)
 
-  >>> # select baselines using the baseline numbers
-  >>> uvd.select(bls=[73736, 73753, 81945])
+  >>> # override a particular parameter
+  >>> rdate_obj = Time(np.floor(uvd1.time_array[0]), format="jd", scale="utc")
+  >>> uvd1.rdate = rdate_obj.strftime("%Y-%m-%d")
+  >>> uvd1.sum_vis(uvd2, inplace=True, override_params=["rdate"])
 
-  >>> # print unique baselines and antennas after select
-  >>> print(np.unique(uvd.baseline_array))
-  [73736 73753 81945]
-  >>> print(list(set(zip(uvd.ant_1_array.tolist(), uvd.ant_2_array.tolist()))))
-  [(8, 25), (4, 25), (4, 8)]
 
-e) Select polarizations
-***********************
-Selecting on polarizations can be done either using the polarization numbers or the
-polarization strings (e.g. "xx" or "yy" for linear polarizations or "rr" or "ll" for
-circular polarizations). If ``telescope.x_orientation`` is set, strings representing
-the physical orientation of the dipole can also be used (e.g. "nn" or "ee).
+UVData: Phasing
+---------------
+Phasing/unphasing data
```

Comment:
I think we need a short paragraph describing the idea of phasing, mentioning the types of phase centers and pointing to the phasing memo for more information.